messaging/publisher/publisher_test.go

```diff
@@ -164,12 +151,7 @@ func (s *ServiceTestSuite) TestTopic() {
 	}
 
 	// We receive the contact code
-	dedupMessage2 := dedup.DeduplicateMessage{
-		DedupID: []byte("1"),
-		Message: message2,
-	}
-
-	err = s.alice.ProcessMessage(dedupMessage2)
+	err = s.alice.ProcessMessage(message2, []byte("1"))
 	s.Require().NoError(err)
```

Comment:
In this line, I am getting:
```
Error:      	Received unexpected error:
                 no payload
```
@cammellos, any idea why is it so? It's a bit strange because it's a public message and `newMessage2, err := s.bob.sendContactCode()` succeeds.

Reply:
That's correct, basically in those messages there's no payload, as we don't transfer any data other than the actual bundles, we should probably not consider it an error as it's a perfectly valid message

Reply:
Ah ok that makes sense. We indeed do not send any message just contact code. The only thing is that it worked before my changes... :D